dotnet/src/Experimental/Agents.UnitTests/Integration/AgentHarness.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+#define DISABLEHOST // Comment line to enable
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Experimental.Agents;
+using Microsoft.SemanticKernel.Experimental.Agents.Internal;
+using Microsoft.SemanticKernel.Experimental.Agents.Models;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SemanticKernel.Experimental.Agents.UnitTests.Integration;
+
+/// <summary>
+/// Dev harness for manipulating agents.
+/// </summary>
+/// <remarks>
+/// Comment out DISABLEHOST definition to enable tests.
+/// Not enabled by default.
+/// </remarks>
+[Trait("Category", "Integration Tests")]
+[Trait("Feature", "Agent")]
+public sealed class AgentHarness
```

Comment:
It feels to me this should be AssistantHarness since this is to exercise the OpenAI assistant functionality

Reply:
I went back and forth on this one. I went with Agent because most of the tests use AgentBuilder and verify things on IAgent.